app/services/exports/lettings_log_export_service.rb

```diff
@@ -219,12 +223,16 @@ def add_scheme_fields!(scheme, attribute_hash)
       attribute_hash["schtype"] = scheme.scheme_type_before_type_cast
       attribute_hash["support"] = scheme.support_type_before_type_cast
       attribute_hash["units_scheme"] = scheme.locations.map(&:units).compact.sum
+      attribute_hash["scheme"] = scheme.id
+      attribute_hash["scheme_status"] = scheme.status_at(attribute_hash["startdate"])
     end
 
     def add_location_fields!(location, attribute_hash)
       attribute_hash["mobstand"] = location.mobility_type_before_type_cast
-      attribute_hash["scheme"] = location.old_visible_id || (location.id + LOG_ID_OFFSET)
+      attribute_hash["scheme_old"] = location.old_visible_id
       attribute_hash["units"] = location.units
+      attribute_hash["location_code"] = location.id
```

Comment:
this is the old core location code right? not the new core local authority code (which we're confusingly calling location_code in our db iirc)

Reply:
I was confused by this too. This is meant to be the location ID that we have on our service

Reply:
Lgtm in that case 👍